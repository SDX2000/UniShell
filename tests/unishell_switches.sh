#!/bin/bash

unishell -c '"Hello A"'
unishell -c 'set msg "Hello B"' -c '$msg'
unishell -c 'set msg "Hello C"; $msg'
unishell -i  --no-banner -c 'exit'
unishell -c 'exit 1'
unishell -c 'exit 2'
