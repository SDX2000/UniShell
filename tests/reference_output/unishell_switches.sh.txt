Hello A
Hello B
Hello C
SYNTAX ERROR:  Expected '-' at position (1, 6) => 'echo *===='.
